spec/features/user_signs_up_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+describe "User sign up" do
+
+  let!(:user) { create(:user) }
+
+  it "with valid email and password" do
+    login("person1@example.com", "password")
+    expect(page).to have_content('Выйти')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.